src/hardware/addon-protocols/can.md

```diff
@@ -4,7 +4,9 @@ permalink: /hardware/addon-protocols/can/
 title: Add-On Protocol - CAN
 --- 
 
-External devices can communicate with the Geotab GO device through the revised Third-Party Data CAN protocol. No initial handshake is required to communicate through the IOX-CAN. The GO device will start processing third-party data if it is in the correct format. Once processed, the third-party data will be saved and sent to MyGeotab as Status Data.
+External devices can communicate with the Geotab GO device through the revised Third-Party Data CAN protocol. The hardware interface will be the [IOX-CAN](https://www.geotab.com/documentation/iox-can/). Two-way communication is supported, allowing a MyGeotab API call to produce messages from the IOX device on the connected CAN network.
+
+No initial handshake is required to communicate through the IOX-CAN. The GO device will start processing third-party data if it is in the correct format. Once processed, the third-party data will be saved and sent to MyGeotab as Status Data.
```

Comment:
The handshake is required if you want to use the data to component to send CAN messages from myG to the external device.

Reply:
@Stephen-F I can update the wording to reflect this. For the IOX-CAN, the handshake would be the external device issuing [Msg 0x81](https://geotab.github.io/sdk/hardware/addon-protocols/can/#msg-type-0x81-third-party-device-id) and getting a [Msg 0x02](https://geotab.github.io/sdk/hardware/addon-protocols/can/#msg-type-0x02-third-party-data-acknowledge) in response, right?

Reply:
yes